nRFCloud/device-simulator-v2
# feat: Add GNSS simulated service

## Summary
Device firmware sends PVT data and we want out simulator to do that.

## Testing
1. Checkout this branch
3. `yarn build`
4. `node dist/cli.js -k <API-key> -h https://api.dev.nrfcloud.com -a -d <DEVICE-ID> -s gnss`
5. Head to dev.nrfcloud.com and the device view
6. Check out the location map, you should see the points showing up around the map.

> [!NOTE]
> Currently the FE has a bug that show's `NaN Speed` for the tooltips, I found it with this change and have a PR on that side to fix it.
